---a/net/minecraft/tileentity/TileEntityDropper.java
+++b/net/minecraft/tileentity/TileEntityDropper.java
@@ -1,17 +1,15 @@
 package net.minecraft.tileentity;
 
-public class TileEntityDropper extends TileEntityDispenser
-{
+public class TileEntityDropper extends TileEntityDispenser {
+
     private static final String __OBFID = "CL_00000353";
 
-    public String getName()
-    {
+    public String getName() {
         return this.hasCustomName() ? this.customName : "container.dropper";
     }
 
-    public String getGuiID()
-    {
+    public String getGuiID() {
         return "minecraft:dropper";
     }
-}
+}
